inst/assets/BS3/pkgdown.js

```diff
@@ -80,7 +80,7 @@
     $(document).ready(function() {
       var copyButton = "<button type='button' class='btn btn-primary btn-copy-ex' type = 'submit' title='Copy to clipboard' aria-label='Copy to clipboard' data-toggle='tooltip' data-placement='left auto' data-trigger='hover' data-clipboard-copy><i class='fa fa-copy'></i></button>";
 
-      $(".examples, div.sourceCode").wrap("<div class='hasCopyButton'></div>");
```

Comment:
not all assets are different but it's easier to have a folder per version I think?

Reply:
Yeah, definitely.